ocrd_network/ocrd_network/helpers.py

```diff
@@ -0,0 +1,42 @@
+from re import split as re_split
+
+
+def verify_database_url(mongodb_address: str) -> str:
+    database_prefix = 'mongodb://'
+    if not mongodb_address.startswith(database_prefix):
+        error_msg = f'The database address must start with a prefix: {database_prefix}'
+        raise ValueError(error_msg)
+    address_without_prefix = mongodb_address[len(database_prefix):]
+    elements = address_without_prefix.split(':', 1)
+    if len(elements) != 2:
+        raise ValueError('The database address is in wrong format')
+    db_host = elements[0]
+    db_port = int(elements[1])
+    mongodb_url = f'{database_prefix}{db_host}:{db_port}'
+    return mongodb_url
+
+
+def verify_and_parse_rabbitmq_addr(rabbitmq_address: str) -> dict:
```

Comment:
Same as above, better use a regular expression, maybe [this one](https://stackoverflow.com/questions/45073320/regex-for-a-url-connection-string).